src/main/groovy/org/pih/warehouse/importer/CycleCountItemsExcelImporter.groovy

```diff
@@ -0,0 +1,68 @@
+package org.pih.warehouse.importer
+
+import grails.util.Holders
+import org.grails.plugins.excelimport.AbstractExcelImporter
+import org.grails.plugins.excelimport.DefaultImportCellCollector
+import org.grails.plugins.excelimport.ExcelImportService
+import org.grails.plugins.excelimport.ExpectedPropertyType
+
+class CycleCountItemsExcelImporter extends AbstractExcelImporter implements DataImporter {
+
+    ExcelImportService excelImportService
+
+    static CELL_REPORTER = new DefaultImportCellCollector()
+
+    static Map COLUMN_MAP = [
+            sheet    : "Sheet1",
+            startRow : 1,
+            columnMap: [
+                    A: "cycleCountId",
+                    B: "productCode",
+                    C: "product.name",
+                    D: "lotNumber",
+                    E: "expirationDate",
+                    F: "binLocation",
+                    G: "quantityCounted",
+                    H: "comment",
+            ]
+    ]
+
+    static Map PROPERTY_MAP = [
+            cycleCountId:             ([expectedType: ExpectedPropertyType.StringType, defaultValue: null]),
+            productCode:                ([expectedType: ExpectedPropertyType.StringType, defaultValue: null]),
+            "product.name":         ([expectedType: ExpectedPropertyType.StringType, defaultValue: null]),
+            "lotNumber":            ([expectedType: ExpectedPropertyType.StringType, defaultValue: null]),
+            "expirationDate":       ([expectedType: ExpectedPropertyType.DateJavaType, defaultValue: null]),
+            "binLocation":          ([expectedType: ExpectedPropertyType.StringType, defaultValue: null]),
+            "quantityCounted":       ([expectedType: ExpectedPropertyType.IntType, defaultValue: null]),
+            "comment":            ([expectedType: ExpectedPropertyType.StringType, defaultValue: null]),
+    ]
```

Comment:
I don't know whether it's just GitHub formatting, but the indentations are not aligned